pkg/ssh/ssh.go

```diff
@@ -305,10 +306,12 @@ func NewSSHTunnelList(user, keyfile string, healthCheckURL *url.URL, stopChan ch
 		defer l.tunnelsLock.Unlock()
 		// Healthcheck each tunnel every minute
 		numTunnels := len(l.entries)
-		for i, entry := range l.entries {
+		i := 0
+		for _, entry := range l.entries {
 			// Stagger healthchecks evenly across duration of healthCheckPoll.
 			delay := healthCheckPoll * time.Duration(i) / time.Duration(numTunnels)
 			l.delayedHealthCheck(entry, delay)
```

Comment:
Change this to `l.delayedHealthCheck(entry, wait.Jitter(healthCheckPoll, 1))` instead of tracking the index.

https://github.com/kubernetes/kubernetes/blob/c516bb51e3e94cd2f5ed5096ca15bb853c38c167/staging/src/k8s.io/apimachinery/pkg/util/wait/wait.go#L186-L197

Reply:
Good idea. :)